Sources/Hedera/Account/AccountId.swift

```diff
@@ -135,6 +135,15 @@ public struct AccountId: Sendable, EntityId, ValidateChecksums {
     public func toBytes() -> Data {
         toProtobufBytes()
     }
+
+    public func populateAccountIdNum(_ client: Client) async throws -> Self {
+        let mirrorNodeGateway = try MirrorNodeGateway.forClient(client)
+        let mirrorNodeService = MirrorNodeService(mirrorNodeGateway)
+
+        let accountNumFromMirror = try await mirrorNodeService.getAccountNum(self.evmAddress!.toString())
+
+        return Self(shard: shard, realm: realm, num: accountNumFromMirror)
+    }
```

Comment:
I really don't like this function name, it's basically "make a version of `self` with the account number from the evm address we have" the function sounds like it's mutating (it isn't).

Reply:
Unused so omitting